src/hydrationRepo.js

```diff
@@ -0,0 +1,11 @@
+class HydrationRepo {
+  constructor(allHydrationData) {
+    this.allHydrationData = allHydrationData;
+  }
+
+  getOuncesByDate(userId, date) {
```

Comment:
This was a the change that had to be made in order to make sure that the user Repository and user instances were created before trying to create instances of activity, hydration and sleep, so that they were in sync. Those classes rely on the users since they put all their data on the user.